### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by praneetmek**

### CODE REVIEW

File: src/main/webapp/scripts/calendar-script.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    if ((dateTime.getHours() % 12) == 0) {
+      time += '12';
+    } else {
+      time += dateTime.getHours() % 12;
+    }
+    time += ':';
+    if (dateTime.getMinutes() < 10) {
+      time += '0' + dateTime.getMinutes();
+    } else {
+      time+=dateTime.getMinutes();
```
Comment (nihabhaskar) [inline src/main/webapp/scripts/calendar-script.js:86]: spaces around +=

File: src/main/webapp/scripts/calendar-script.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    return time;
+  }
+
+  const startTime = getTime(start);
+  const endTime = getTime(end);
+
+  if (startDate == endDate) {
+    timeNode.appendChild(
+        document.createTextNode(
+            startDate+ ' ' + startTime + ' - ' + endTime));
```
Comment (nihabhaskar) [inline src/main/webapp/scripts/calendar-script.js:97]: space around +

File: src/main/webapp/scripts/calendar-script.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  }).then(function() {
+    listElement.appendChild(button);
+  });
+  $('#open-events').append(listElement);
+}
+
+/**
+ * Adds an event to calendar with given id
+ * @param {Event} eventToAdd Event to be added
+ * @param {String} calendarID ID of calendar that is targeted
```
Comment (nihabhaskar) [inline src/main/webapp/scripts/calendar-script.js:145]: so each even has a unique calendar ID?

File: src/main/webapp/scripts/calendar-script.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    const events = response.result.items;
+    return events;
+  });
+}
+
+/**
+ * Adds an event to the webpage
+ * @param {Event} event Event to add
+ */
+function addEventToPage(event) {
```
Comment (nihabhaskar) [inline src/main/webapp/scripts/calendar-script.js:45]: where do events get added bottom of list or top on page

Comment (praneetmek) [reply src/main/webapp/scripts/calendar-script.js:44]: in the ul

File: src/main/webapp/scripts/calendar-script.js (hunk 3 above)
Comment (praneetmek) [reply src/main/webapp/scripts/calendar-script.js:144]: calendar ID is the ID of the calendar you wanna add the event to

File: src/main/webapp/scripts/calendar-script.js (hunk 4 above)
Comment (nihabhaskar) [reply src/main/webapp/scripts/calendar-script.js]: should it be organized by like time like soonest to latest

File: src/main/webapp/scripts/calendar-script.js (hunk 3 above)
Comment (nihabhaskar) [reply src/main/webapp/scripts/calendar-script.js]: ah ok

File: src/main/webapp/scripts/calendar-script.js (hunk 4 above)
Comment (praneetmek) [reply src/main/webapp/scripts/calendar-script.js]: Ya it is ordered in that way

**Pull request merged**